server/events/command_runner.go

```diff
@@ -145,10 +146,15 @@ func (c *DefaultCommandRunner) RunAutoplanCommand(baseRepo models.Repo, headRepo
 		return
 	}
 
+	err := c.PreWorkflowHooksCommandRunner.RunPreHooks(ctx)
+
+	if err != nil {
+		ctx.Log.Err("Error running pre-workflow hooks %s. Proceeding with %s command.", err, models.PlanCommand)
+	}
+
 	autoPlanRunner := buildCommentCommandRunner(c, models.PlanCommand)
 	if autoPlanRunner == nil {
 		ctx.Log.Err("invalid autoplan command")
-		return
```

Comment:
Don't you want this? Otherwise go will panic.

Reply:
oh i think this can be removed since we panic if we can't get it from the map in the function itself:

https://github.com/lyft/atlantis/blob/nish/0rca-665/server/events/command_runner.go#L84:L85

Reply:
Do we even need this check then?